app/javascript/courses/review/types/CoursesReview__SubmissionDetails.res

```diff
@@ -4,6 +4,7 @@ module Student = CoursesReview__Student
 module ReviewChecklistItem = CoursesReview__ReviewChecklistItem
 module SubmissionMeta = CoursesReview__SubmissionMeta
 module Coach = UserProxy
+module Reviewer = UserProxy
```

Comment:
It might make sense to make this a module type here...

```rescript
module Reviewer {
  type t = {
    user: UserProxy.t,
    assignedAt: Js.Date.t
  }

  // let make = (~user, ~assignedAt) => { ... }

  let read = t => (t.user, t.assignedAt);
}
```

...so that these two properties are tied together as a single optional in the type system.